### ISSUE TITLE
Build failed in Xcode 7

### ISSUE THREAD

**Issue by omnicode** [don/BluetoothSerial#154]
Create project "Hello world", add plugin and try to build.

../Plugins/com.megster.cordova.bluetoothserial/BluetoothSerial/CBPeripheral+Extensions.m:29:15: No visible @interface for 'CBPeripheral' declares the selector 'UUID'
../Plugins/com.megster.cordova.bluetoothserial/BluetoothSerial/CBPeripheral+Extensions.m:30:82: Property 'UUID' not found on object of type 'CBPeripheral *'

**Comment by mounir01** [comment]
Hi, I have the same problem as you. did you find a solution to this?

**Comment by omnicode** [comment]
Nope, just ask my friend - ios developer for help. Waiting for his answer. I he find the solution give you a notify here.

**Comment by mounir01** [comment]
thanks

**Comment by sandeepmistry** [comment]
@omnicode @mounir01 you can try changing the following lines from:

```
    NSString *uuidString = NULL;
    if ([self UUID]) {
        uuidString = (__bridge_transfer NSString *)CFUUIDCreateString(NULL, self.UUID);
    } else {
        uuidString = @"";
    }
```

to

```
    NSString *uuidString = NULL;
    if ([self identifier]) {
        uuidString = (__bridge_transfer NSString *)CFUUIDCreateString(NULL, self.identifier);
    } else {
        uuidString = @"";
    }
```

https://github.com/don/BluetoothSerial/blob/master/src/ios/CBPeripheral%2BExtensions.m#L28

```UUID``` was deprecated in iOS7, so might be removed in iOS9 ... 

* https://developer.apple.com/library/ios/documentation/CoreBluetooth/Reference/CBPeripheral_Class/index.html#//apple_ref/occ/instp/CBPeripheral/UUID
* https://developer.apple.com/library/ios/documentation/CoreBluetooth/Reference/CBPeripheral_Class/index.html#//apple_ref/occ/instp/CBPeripheral/identifier

**Comment by mounir01** [comment]
tried this, but it's giving me this error: incompatible pointer types passing retainable parameter of type 'NSUUID*' to CF function expecting 'CFUUIDRef' (aka 'const struct_CFUUID*') type

**Comment by sandeepmistry** [comment]
@mounir01 second attempt try:

```
    NSString *uuidString = self.identifier.UUIDString;
```

**Comment by mounir01** [comment]
that's what i get:
![capture](https://cloud.githubusercontent.com/assets/8737377/9234805/14ae20aa-4109-11e5-8276-820cd0fffb4e.PNG)

**Comment by sandeepmistry** [comment]
@mounir01 sorry, I mean't replace the entire ``if`` statement with that line ...

```
    NSString *uuidString = NULL;
    if ([self UUID]) {
        uuidString = (__bridge_transfer NSString *)CFUUIDCreateString(NULL, self.UUID);
    } else {
        uuidString = @"";
    }
```

becomes:

```
NSString *uuidString = self.identifier.UUIDString;
```

**Comment by mounir01** [comment]
thank's lot, it works.
I stell have a similar problem with uuid in MEGBluetoothSerial. please can you help me with that.
my problem is:
![capture 1](https://cloud.githubusercontent.com/assets/8737377/9235168/109b0bfc-410b-11e5-87b3-3daf347bd2bc.PNG)

**Comment by sandeepmistry** [comment]
Similar change:

```
        if (p.UUID != NULL) {
            // Seriously WTF?
            CFStringRef s = CFUUIDCreateString(NULL, p.UUID);
            NSString *uuid = [NSString stringWithCString:CFStringGetCStringPtr(s, 0)
                                                encoding:(NSStringEncoding)NSUTF8StringEncoding];
            [peripheral setObject: uuid forKey: @"uuid"];
            [peripheral setObject: uuid forKey: @"id"];
        }
        else {
            [peripheral setObject: @"" forKey: @"uuid"];
        }
```

to

```
        NSString *uuid = p.identifier.UUIDString;
        [peripheral setObject: uuid forKey: @"uuid"];
        [peripheral setObject: uuid forKey: @"id"];
```

**Comment by mounir01** [comment]
thank's sandeepmistry it works

**Comment by omnicode** [comment]
I am still recieve strange errors

Undefined symbols for architecture x86_64:
  "_CBAdvertisementDataManufacturerDataKey", referenced from:
      -[CBPeripheral(com_megster_rfduino_extension) setAdvertisementData:RSSI:] in CBPeripheral+Extensions.o
  "_OBJC_CLASS_$_CBPeripheral", referenced from:
      l_OBJC_$_CATEGORY_CBPeripheral_$_com_megster_rfduino_extension in CBPeripheral+Extensions.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by omnicode** [comment]
Problem solved. Thanks!

**Comment by don** [comment]
Thanks @sandeepmistry 

I'll get these changes rolled into a future version

**Comment by don** [comment]
Fixed by #161

**Comment by halygo** [comment]
Hello @sandeepmistry 

I have a littel problem : NSString *uuid = p.identifier.UUIDString; Use  of undeclared identifier 'p' 

can you help me

Thanks
